cmd/pop/eval.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"github.com/go-clix/cli"
+	"github.com/pollypkg/polly/pkg/pop"
+)
+
+func mixCmd() *cli.Command {
+	cmd := &cli.Command{
+		Use:   "mix <path>",
+		Short: "output pop in mixin compatible format",
+		Args:  cli.ArgsExact(1),
+	}
+
+	printer := cmd.Flags().StringP("output", "o", "json", "output format. One of json, yaml")
+	system := cmd.Flags().StringP("system", "s", "", "choose subsystem. One of alerts, rules, grafana (default all)")
+
+	cmd.Run = func(cmd *cli.Command, args []string) error {
+		p, err := pop.Load(args[0])
+		if err != nil {
+			return err
+		}
+
+		mix, err := p.Mixin()
+		if err != nil {
+			return err
+		}
+
+		var out interface{} = mix
```

Comment:
You could inline choosePrinter and not have to use `interface{}` anymore, if I'm not mistaken.